crates/cornucopia/src/container.rs

```diff
@@ -5,7 +5,7 @@ use self::error::Error;
 /// Starts Cornucopia's database container and wait until it reports healthy.
 pub fn setup(podman: bool) -> Result<(), Error> {
     spawn_container(podman)?;
-    healthcheck(podman, 120, 50)?;
```

Comment:
But it retries the pings, why would it be a problem that the container isn't ready yet?

Reply:
I think the health checker is erroring out before it could do any retry:

```rust
fn healthcheck(podman: bool, max_retries: u64, ms_per_retry: u64) -> Result<(), Error> {
    let slow_threshold = 10 + max_retries / 10;
    let mut nb_retries = 0;
    while !is_postgres_healthy(podman)? { // <------ this won't execute for me
       
    }
    // ...
}
```

I haven't confirmed this but a quick Look suggests that `is_postgres_healthy` is calling a command on the `cornucopia_postgres` container before it's even created:


```rust
/// Checks if Cornucopia's container reports healthy
fn is_postgres_healthy(podman: bool) -> Result<bool, Error> {
    Ok(cmd(
        podman,
        &["exec", "cornucopia_postgres", "pg_isready"],
        "check container health",
    )
    .is_ok())
}
```

Reply:
But `is_postgres_healthy` can only return an `Ok(true)` or `Ok(false)`, so it cannot actually error out?